src/main/telemetry/smartport.c

```diff
@@ -588,7 +588,7 @@ void handleSmartPortTelemetry(void)
                         vfasVoltage = vbat / batteryCellCount;
                     else
                         vfasVoltage = vbat;
-                    smartPortSendPackage(id, vfasVoltage * 10); // given in 0.1V, convert to volts
```

Comment:
I'm pretty sure the comment is wrong. If `vfasVoltage = vbat` and vbat resolution is 100mV then `vfasVoltage * 10` gives result in 10mV increments not in 1V increments.